tests/integration/test_zos_files.py

```diff
@@ -46,24 +50,24 @@ def test_list_dsn_should_return_a_list_of_datasets(self):
             command_output = self.files.list_dsn(self.files_fixtures["TEST_HLQ"], scenario["attributes"])
 
             # Assert that command_output['items'] is a list
-            self.assertIsInstance(command_output["items"], list)
+            self.assertIsInstance(command_output, DatasetListResponse)
 
             # Assert that command_output['items'] contains at least one item
             self.assertGreater(len(command_output["items"]), 0)
 
             # Assert that the first item in the list has 'dsname' defined
             first_item = command_output["items"][0]
-            self.assertIn("dsname", first_item)
+            self.assertTrue(hasattr(first_item, "dsname"))
 
             # Assert that the first item in the list has the expected attributes defined
-            attributes = first_item.keys()
+            attributes = dir(first_item)
```

Comment:
It might be nice if we implemented standard dictionary methods like `.keys`, `.values` etc on our response data classes, but this could be done as a future enhancement 😋